src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -194,6 +194,8 @@ fields specific to anatomical scans:
 | **Key name**            | **Requirement level** | **Data type** | **Description**                                                                                                                          |
 | ----------------------- | --------------------- | ------------- | ---------------------------------------------------------------------------------------------------------------------------------------- |
 | ContrastBolusIngredient | OPTIONAL              | [string][]    | Active ingredient of agent. Values MUST be one of: IODINE, GADOLINIUM, CARBON DIOXIDE, BARIUM, XENON Corresponds to DICOM Tag 0018,1048. |
+| RepetitionTimeExcitation | OPTIONAL              | [number][]    | The time in seconds between successive excitation pulses that excite a spin system. The DICOM tag that best refers to this parameter is [(0018, 0080)](http://dicomlookup.com/lookup.asp?sw=Tnumber&q=(0018,0080)). This field may be used together with the `RepetitionTimePreparation` for certain use cases, such as [MP2RAGE](https://infoscience.epfl.ch/record/172927/files/mp2rage.pdf). Use `RepetitionTimeExcitation` (in combination with `RepetitionTimePreparation` if needed) for anatomy imaging data rather than `RepetitionTime` as it is already defined as the amount of time that it takes to acquire a single volume in the [task imaging data](#task-including-resting-state-imaging-data) section.|
```

Comment:
```
[MP2RAGE](https://infoscience.epfl.ch/record/172927/files/mp2rage.pdf)
```

Detail I am not sure about because I don't think it is consistent through the specs, but isn't it better to have a DOI for references? 

I think we sort of have in mind to automate the "bibliography" part of the specs, no?

Reply:
Sure @Remi-Gau, I will replace them with DOI URLs.

Reply:
1a9412c32bdb6aa14314b1e0df2cb63c70a81e59